src/issues.js

```diff
@@ -89,13 +89,12 @@ var mergeNewIssues = function (newIssues, cacheIssues) {
 }
 
 var getCacheContents = function () {
-
-
-    var repo_name = (/^.*\/([^\/]+)$/).exec(gUrl).slice(1).toLowerCase();

```

Comment:
Hmm, didn't even check, I think that was something I got off StackOverflow... Thanks for catching